src/models/modules/discriminator.py

```diff
@@ -128,29 +128,50 @@ def __init__(self) -> None:
         self.logits_uncond = UnconditionalLogits()
         self.logits_cond = ConditionalLogits()
 
-    def forward(
-        self, images: torch.Tensor, word_embs: torch.Tensor, sent_embs: torch.Tensor
+    def get_cond_logits(
+        self, visual_features: torch.Tensor, sent_embs: torch.Tensor
     ) -> Any:
+        """
+        Compute logits for conditional adversarial loss
+
+        :param torch.Tensor visual_features: Features from Inception encoder. Bx128x17x17
+        :param torch.Tensor sent_embs: Sentence embeddings from text encoder. Bx256
+        :return: Logits for conditional adversarial loss. Bx1
+        :rtype: Any
+        """
+        return self.logits_cond(visual_features, sent_embs)
+
+    def get_uncond_logits(self, visual_features: torch.Tensor) -> Any:
```

Comment:
Wrapper functions `get_cond_logits`, `get_word_level_logits`, and `get_uncond_logits` are useless because you can just call logit extractors directly. They have the same notation even, no need to make a function that just calls one function with same params

Reply:
chaned this, review and resolve please.